pages/index.tsx

```diff
@@ -82,20 +79,18 @@ const Entry = ({ doc }) => {
         <h3>{doc.title}</h3>
       </Link>
       <p>Posted on {new Date(doc.pubDate).toLocaleDateString()}</p>
-      <p
-        dangerouslySetInnerHTML={{
-          __html: doc.content.replace(/^"/, '').replace(/"$/, '')
-        }}
-      ></p>
+      <p>
+        {excerpt(doc.contentSnippet?.replace(/^"/, '').replace(/"$/, ''), 50)}
+      </p>
       <div className={styles.readMoreArea}>
         <Link href={`/read?url=${doc.link}`}>
-          <Button highlight>Read more...</Button>
+          <Button highlight>Read more</Button>
         </Link>
-        <Link href={doc.link}>
+        <a href={doc.link} target="_blank" rel="noopener noreferrer">
           <Button iconRight icon={externalLink}>
             Original link
           </Button>
-        </Link>
+        </a>
```

Comment:
Following moai document, `Button` has `href` props to render button as `a` tag. Following w3, the element `button` must not appear as a descendant of the `a` element.

Reply:
Alright, I have fixed in the next PR but not this one, let me move the fix to this PR as well.

Reply:
:heavy_check_mark: